src/endpoint/s3/s3_bucket_logging.js

```diff
@@ -0,0 +1,52 @@
+/* Copyright (C) 2016 NooBaa */
+'use strict';
+
+const dbg = require('../../util/debug_module')(__filename);
+const http_utils = require('../../util/http_utils');
+
+function is_bucket_logging_enabled(source_bucket) {
+
+    if (!source_bucket || !source_bucket.bucket_info.logging) {
+
+        return false;
+    }
+
+    return true;
+
+}
+
+function endpoint_bucket_op_logs(op_name, req, res, source_bucket) {
+
+    dbg.log2("Sending op logs for op name = ", op_name);
+    // 1 - Get all the information to be logged in a log message.
+    // 2 - Format it and send it to log bucket/syslog.
+
+    const s3_log = get_bucket_log_record(op_name, source_bucket, req, res);
+    dbg.log1("!!!BUCKET Operation Logs!!! ", s3_log);
```

Comment:
A bit dramatic...

```suggestion
    dbg.log1("bucket op log = ", s3_log);
```

Also, consider prefixing all logs realted to bucket op logs with a common prefix.
It would make it easier to grep for them.
Eg-
dbg.log("bucket op log: sending op logs for op name = ")